web/app/js/components/Sidebar.jsx

```diff
@@ -18,7 +19,13 @@ class Sidebar extends React.Component {
     this.loadFromServer = this.loadFromServer.bind(this);
     this.handleApiError = this.handleApiError.bind(this);
 
-    this.state = {
+    this.state = this.getInitialState();
+  }
+
+  getInitialState() {
+    return {
+      pollingInterval: 12000,
```

Comment:
did you mean `pollingInterval: 2000` ?

Reply:
Nah, I wanted it to be 12 seconds. I don't think the number of namespaces will change that rapidly, so I didn't think polling every 2 seconds would be necessary.